plugins/quickicon/eos/src/Extension/Eos.php

```diff
@@ -34,12 +34,12 @@ final class Eos extends CMSPlugin implements SubscriberInterface
     use DatabaseAwareTrait;
 
     /**
-     * The EOS date for 4.4.
+     * The EOS date for 5.x.
      *
      * @var    string
      * @since 4.4.0
      */
-    private const EOS_DATE = '2025-10-17';
+    private const EOS_DATE = '2027-11-17';
```

Comment:
```suggestion
    private const EOS_DATE = '2027-10-19';
```
No reason that the EOS date of 5.x is one month longer then planed.

Reply:
@HLeithner The project road map says that here: https://developer.joomla.org/roadmap.html . I have not invented this date. I'm happy to change it if the project road map page is changed before that.

Reply:
The roadmap has also my name spelled incorrectly ;-)

Reply:
I fixed the roadmap

Reply:
Well you are the RL but it's not my job to change the road map. I'll accept the suggestion here but if someone complains that the date is wrong referring to the road map then someone else shall fix that.